pkgs/modules/python/default.nix

```diff
@@ -94,6 +94,10 @@ let
 
   poetry-wrapper = pythonWrapper { bin = "${poetry}/bin/poetry"; name = "poetry"; };
 
+  python-wrapped = pkgs.callPackage ../../python-wrapped {
```

Comment:
python-wrapper vs python-wrapped seems too subtly different. Can we make them more different?

Reply:
Yes, no problem. My intention was to actually delete `python-wrapper` entirely, but it was too deeply entwined with the prybar pythons.